packages/paste-website/src/__tests__/packageStatusLegend.test.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import {render, screen} from '@testing-library/react';
+import {Theme} from '@twilio-paste/theme';
+import {PackageStatusLegend} from '../components/shortcodes/package-status-legend';
+
+describe('PackageStatusLegend', () => {
```

Comment:
Praise: thanks for adding tests about for logic!

Reply:
I was so sick of having to trace it through every time. No more!